frontend/public/lang/default/de.json

```diff
@@ -277,6 +280,7 @@
             "warriorLeader": "Chef",
             "finalPoints": "Finale Punkte",
             "points": "Punkte",
+            "planSkipped": "Plan skipped.",
```

Comment:
```Plan übersprungen``` if you could please, also remove the period in all cases.

Reply:
Done